### ISSUE TITLE
Author field problems

### ISSUE THREAD

**Issue by krlmlr** [metacran/description#22]
- Authors@R messed up by normalize()
- message suggests using to_authors_at_r() function which doesn't seem to exist
- `description::description$new("!new")$get_author()` returns NULL but shouldn't

In 6dfb24ee786f.

NB: Have you tried as.person.character() for parsing Author and Maintainer fields?

**Comment by gaborcsardi** [comment]
> - Authors@R messed up by normalize()

That's subjective. :) I think it looks nice. Although the missing key and the empty line is a big problem indeed. I wonder how I missed this.....

```
    c(person(given = "Hadley",
             family = "Wickham",
             role = c("aut", "cre", "cph"),
             email = "hadley@rstudio.com"),
      person(given = "Peter",
             family = "Danenberg",
             role = c("aut", "cph"),
             email = "pcd@roxygen.org"),
      person(given = "Manuel",
             family = "Eugster",
             role = c("aut", "cph")),
      person(given = "RStudio",
             role = "cph"))
```

> - message suggests using to_authors_at_r() function which doesn't seem to exist

Yes, I wanted to write it, but gave up in the end. Too hard to parse the field(s) for too little gain. I'll just remove the message for now I think.

> - `description::description$new("!new")$get_author()` returns NULL but shouldn't

Yes, probably `!new` should create a file with `Authors@R`. TBH I hate `Authors@R`, because it looks so different than the rest of the file, but I guess that's the standard now.

> NB: Have you tried as.person.character() for parsing Author and Maintainer fields?

Hmmm. In 3.2.4 there is no such function, but `as.person.character` might do the trick. I missed that.

**Comment by krlmlr** [comment]
It's as.person.default(). The formatting for a single Author@R looked odd to me.

**Comment by gaborcsardi** [comment]
> It's as.person.default(). 

That's it. I tried it, it messes up things even with simple inputs. So people can convert to `Authors@R` manually I guess.

> The formatting for a single Author@R looked odd to me.

Well, it is the standard formatting, I just use `format()`. Since we don't ever reformat, unless the user tesll us to, I think this is OK. It is different from the Hadley style, but I think it looks OK.

**Comment by gaborcsardi** [comment]
The rest I fixed I believe. Thanks!
